lib/langchain/llm/unified_parameters.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+module Langchain::LLM
+  class UnifiedParameters
```

Comment:
@codenamev Think we'd jam all of the parameters for `chat()`, `embed()` and `complete()` under `UnifiedParameters` or we'd introduce individual classes for `UnifiedChatParameters`, `UnifiedEmbedParameters`, `UnifiedCompleteParameters`?

Reply:
This makes a lot of sense.  I originally had a `LLM::Parameters::Chat`, but removed it because it was just the schema hash.  Let me refactor it back.  It will certainly make documenting things easier.